dotcms-integration/src/test/java/com/dotcms/contenttype/test/ContentTypeAPIImplTest.java

```diff
@@ -2809,4 +2809,93 @@ public void testCopyFromAndDependenciesToAnotherSite() throws DotDataException,
+	 *     Fields.</li>
+	 *     <li><b>Expected Result: </b>The copied Content Type must point to the same Workflow
+	 *     Scheme as it was copied with dependencies, and also point to the new Site. Moreover, NO
+	 *     Relationship Fields must be present.</li>
+	 * </ul>
+	 */
+	@Test
+	public void testCopyFromAndDependenciesToAnotherSiteWithNoRelationshipFields() throws DotDataException, DotSecurityException {
+		// ╔══════════════════╗
+		// ║  Initialization  ║
+		// ╚══════════════════╝
+		final ContentTypeAPI contentTypeAPI = APILocator.getContentTypeAPI(APILocator.systemUser());
+		final WorkflowAPI workflowAPI = APILocator.getWorkflowAPI();
+		final long millis = System.currentTimeMillis();
+		final String siteTwoName = "sitetwo" + millis +".com";
+		ContentType sourceContentType = null;
+		ContentType copiedContentType = null;
+		WorkflowScheme workflowScheme = null;
+		try {
+			// ╔════════════════════════╗
+			// ║  Generating Test data  ║
+			// ╚════════════════════════╝
+			final List<Field> fields = new ArrayList<>();
+			fields.add(new FieldDataGen().name("Title").velocityVarName("title").next());
+			fields.add(new FieldDataGen().type(RelationshipField.class)
+					.name("First Related Content").velocityVarName("firstRelatedContent").relationType("webPageContent").values("0").next());
+			fields.add(new FieldDataGen().name("Author").velocityVarName("author").next());
+			fields.add(new FieldDataGen().type(WysiwygField.class).name("Description").velocityVarName("description").next());
+			fields.add(new FieldDataGen().type(RelationshipField.class)
+					.name("Second Related Content").velocityVarName("secondRelatedContent").relationType("webPageContent").values("0").next());
+			sourceContentType = new ContentTypeDataGen()
+					.host(APILocator.systemHost()).description("This is a description of my test Content Type")
+					.fields(fields)
+					.nextPersisted();
+			sourceContentType = contentTypeAPI.save(sourceContentType);
+			final Host siteTwo = new SiteDataGen().name(siteTwoName).nextPersisted();
+			workflowScheme = new WorkflowDataGen().name("Test Workflow_" + millis).nextPersisted();
+			final Set<String> schemesIds = new HashSet<>();
+			schemesIds.add(workflowScheme.getId());
+			workflowAPI.saveSchemeIdsForContentType(sourceContentType, schemesIds);
+			final String newVariableName = sourceContentType.variable() + millis;
+			// Pass down the flag to skip copying Relationship fields
+			copiedContentType =
+					contentTypeAPI.copyFromAndDependencies(
+							new CopyContentTypeBean.Builder().sourceContentType(sourceContentType).name(newVariableName).newVariable(newVariableName).build(),
+							siteTwo,
+							false);
+
+			// ╔══════════════╗
+			// ║  Assertions  ║
+			// ╚══════════════╝
+			assertNotEquals("Both Content Types MUST belong to different Sites", sourceContentType.siteName(), copiedContentType.siteName());
```

Comment:
Here we can assert we have the sites we are expecting instead of asserting they are different from each other

Reply:
Done!